payloads/v10/channel.ts

```diff
@@ -2305,6 +2345,12 @@ export interface APIContainerComponent extends APIBaseComponent<ComponentType.Co
 	components: APIComponentInContainer[];
 }
 
+export interface APILabelComponent extends APIBaseComponent<ComponentType.Label> {
+	label: string;
+	description?: string;
+	component: APIComponentInModalLabel;
```

Comment:
I'd probably do APIComponentInLabel, not InModalLabel, should discord ever make this component work outside modals too